tests/services/test_alerts.py

```diff
@@ -182,88 +182,59 @@ def test_get_details_when_observation_data_not_parseable_remains_unchanged(
         expected_observation_data = '{"invalid_json": ][ }'
         assert observation_data == expected_observation_data
 
-    def test_resolve_when_not_given_tenant_id_posts_expected_data(
```

Comment:
There should be a test that makes sure when you pass `None` for the`note` parameter, that it actually uses the empty string

Reply:
Done.